src/network/internet_gaming.cc

```diff
@@ -974,10 +978,9 @@ void InternetGaming::send(const std::string& msg) {
 			m.string(arg);
 			net->send(m);
 			return;
-		} else {
-			// let everything else pass
-			goto normal;
 		}
+		// let everything else pass
+		goto normal;
 	} else {
```

Comment:
as far as I can see we are returning or going to normal in all the prvious if's so this could be dropped as well?

Reply:
The first `if` block from line 881 does not return, so when it's true then the `else` block is skipped.